pkg/query/version_diff_test.go

```diff
@@ -0,0 +1,62 @@
+package query
+
+import (
+	"testing"
+
+	"github.com/Jguer/yay/v12/pkg/text"
+)
+
+func TestVersionDiff(t *testing.T) {
```

Comment:
I think you have to force `text.Color = true` and then unset it at the end of the test to stop the automatic tty detection from turning off colors.

One of the unfortunate places where we haven't gotten rid of globals

Reply:
Added the change.